README.md

```diff
@@ -0,0 +1,117 @@
+# Web Test Transformer
+
+- [Web Test Transformer](#web-test-transformer)
+  - [Overview](#overview)
+  - [Setup](#setup)
+  - [Usage](#usage)
+    - [Dry Run](#dry-run)
+    - [CLI Flags](#cli-flags)
+      - [Required Flags](#required-flags)
+  - [Verification](#verification)
+    - [All PASS Requirement](#all-pass-requirement)
+  - [Issues](#issues)
+
+## Overview
+Web Test Transformer is designed to transform HTML tests in Chromium source from the legacy
+[js-test.js](https://source.chromium.org/chromium/chromium/src/+/master:third_party/blink/web_tests/resources/js-test.js)
+framework to the current
+[testharness.js](https://source.chromium.org/chromium/chromium/src/+/master:third_party/blink/web_tests/resources/testharness.js)
+framework.
+
+
+View the design document at [go/web-test-transformer](go/web-test-transformer) (internal Google access only).
```

Comment:
github doesn't understand go-links, so you probably want http://go/web-test-transformer